docs/advanced/api_examples.yml

```diff
@@ -208,6 +208,43 @@
 
 -
   id:
+    "delete dataset"
+
+  doc: |
+    Delete a dataset.
+
+  requires:
+    - "create dataset with dataset_id"
+
+  request: |
+    DELETE /v1/configuration/datasets/886ed03a-5606-453a-94a9-a1cbaf35164c HTTP/1.1
+
+    {"primary": "%(NODE_1)s"}
```

Comment:
I don't think client supplies a request body. Perhaps this is a copy / paste error.